libs/components/router/src/lib/modules/href/href.directive.spec.ts

```diff
@@ -1,5 +1,4 @@
-import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { StaticProvider } from '@angular/core';
+import { Provider } from '@angular/core';
```

Comment:
👇 The test expectations should be the same as they were before. However, the way these tests were setup originally made it difficult to test the new behavior, so I refactored the `setup` function to be more helpful.